src/structures/Realm.js

```diff
@@ -0,0 +1,14 @@
+module.exports = class Realm {
+  constructor (api, data) {
+    this.api = api
+    Object.assign(this, data)
```

Comment:
Let's be explicit here with all the fields we expect to return as part of Realm

Reply:
Sure

Reply:
What do we do here if both APIs have separate responses?

Reply:
That’s the point of the library, to abstract whatever the server is responding with. If there is functionally one value with two different names, pick one of them for the library API

for example
WorldName vs worldName => pick the latter one, assign the servers variable to that name on both editions

Reply:
Sorry I understand that part, my question is worded awfully, what I'm getting at is Javas API and Bedrocks API have certain features that the other doesn't, for example, the Bedrock API allows you to manage players, set/get content packs and a handful of other things not present on Java. So now say we have a `getBlocklist()` fn on the Realms class which can only be functional on the bedrock instance, how would you recommend we handle it on a java instance?

Reply:
Go through the APIs and if there is an analog between two different features you can expose one API for things that exist exclusively on on edition without any analog you can expose a function under the Bedrock/Java API classes, like in my example above

Reply:
Right gotcha, I've now extended the RealmsAPI from the platforms API so this is possible, like in the example